package.json

```diff
@@ -608,18 +608,28 @@
                 },
                 {
                     "command": "kubernetes.cloudExplorer.mergeIntoKubeconfig",
-                    "group": "7",
+                    "group": "7@1",
                     "when": "view == kubernetes.cloudExplorer && viewItem =~ /kubernetes\\.providesKubeconfig/i"
                 },
                 {
                     "command": "kubernetes.cloudExplorer.saveKubeconfig",
-                    "group": "7",
+                    "group": "7@4",
                     "when": "view == kubernetes.cloudExplorer && viewItem =~ /kubernetes\\.providesKubeconfig/i"
                 },
                 {
                     "command": "kubernetes.cloudExplorer.findProviders",
                     "when": "view == kubernetes.cloudExplorer && viewItem == kubernetes.noProviders",
                     "group": "1"
+                },
+                {
+                    "command": "extension.ConfigureKubernetesPipeline",
+                    "group": "7@2",
+                    "when": "view == kubernetes.cloudExplorer && viewItem =~ /kubernetes\\.providesKubeconfig/i"
```

Comment:
do have any dependency on `.providesKubeconfig`? If no, then it should not be in the condition. On the other hand, should there be a check for Azure cloud provider?

Reply:
Removed the dependency on kubeConfig. Not sure, how to put a condition on cloud provider.
Tried
``` kubernetes.cloudExplorer.provider.cloudName == 'Azure' ``` and some other options as well, didn't work.
Will have to check for the option